src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.cs

```diff
@@ -472,14 +472,29 @@ private void QueueApplyReplacements()
             }
 
             var asyncToken = _asyncListener.BeginAsyncOperation(nameof(QueueApplyReplacements));
-            _conflictResolutionTask.SafeContinueWith(
-                t => ApplyReplacements(t.Result, _conflictResolutionTaskCancellationSource.Token),
-                _conflictResolutionTaskCancellationSource.Token,
-                TaskContinuationOptions.OnlyOnRanToCompletion,
-                ForegroundTaskScheduler).CompletesAsyncOperation(asyncToken);
+            _conflictResolutionTask
```

Comment:
In general i would prefer to see ContinueWith's replaced with 'await' in async methods/local-functions.  For example, this seems like it would be easier to write as something akin to:

```c#
FooAsync(asyncToken, _conflictResolutionTaskCancellationSource.Token);
...
private async void FooAsync(IAsyncToken token, CancellationToken ct)
{
    var t1 = await _conflictResolutionTask;
    var t2 = ApplyResults(t1, ct);
    var t3 = await ComputeMergeResultAsync(t2, ct);
    await SwitchToUIThread();
    await ApplyReplacements(...., ct);
}
```